tests/integration/nautobot-populate.py

```diff
@@ -632,6 +632,17 @@ def make_nautobot_calls(endpoint, payload):
     contacts = [{"name": "My Contact"}, {"name": "My Contact 2"}]
     created_contacts = make_nautobot_calls(nb.extras.contacts, contacts)
 
+    # Create Controller
+    controller = [
+        {"name": "controller_one", "location": "Child Test Location", "status": {"name": "Active"}},
+        {"name": "controller_two", "location": "Child Test Location", "status": {"name": "Active"}},
```

Comment:
@joewesch please give me a hint why this is not working.

Reply:
Your problem is here:
```python
created_controller = make_nautobot_calls(nb.extras.controller, controller)
```

Controllers aren't in the `extras` app, but rather the `dcim` app. Plus, the endpoint is plural (controller**s**):
```python
created_controller = make_nautobot_calls(nb.dcim.controllers, controller)
```